src/main/charts/jira/values.yaml

```diff
@@ -461,6 +461,19 @@ jira:
     #
     secretKey: license-key
 
+  shutdown:
```

Comment:
For consistency can we:

- include a general comment on what this `shutdown` stanza controls 
- leave a carriage return after `shutdown`